crux-bench/src/crux/bench/main.clj

```diff
@@ -11,15 +11,21 @@
   (bench/post-to-slack (format "*Starting Benchmark*, Crux Version: %s, Commit Hash: %s\n"
                                bench/crux-version bench/commit-hash))
 
-  (bench/with-nodes [node]
-    (let [devices-results (devices/run-devices-bench node)
-          weather-results (weather/run-weather-bench node)
-          watdiv-results (watdiv-crux/run-watdiv-bench node {:test-count 100})
-          result-messages [(bench/results->slack-message devices-results :ts-devices)
-                           (bench/results->slack-message weather-results :ts-weather)
-                           (bench/results->slack-message
-                            [(first watdiv-results) (watdiv-crux/->query-result (rest watdiv-results))]
-                            :watdiv-crux)]]
-      (bench/send-email-via-ses (string/join "\n" result-messages))))
+  (->> (bench/with-nodes [node]
+         (let [devices-results (devices/run-devices-bench node)
```

Comment:
it looks like this `let` isn't yielding any value, so I'd be surprised if this worked?

Reply:
let's also try to split out the part that runs the bench vs the part that outputs to slack/email, it seems they're a little convoluted here. maybe the `let` could return the results out of the with-nodes, then we can process the results outside?